Userland/Libraries/LibCpp/AST.h

```diff
@@ -144,7 +145,7 @@ class InvalidDeclaration : public Declaration {
 
 public:
     virtual ~InvalidDeclaration() override = default;
-    virtual const char* class_name() const override { return "InvalidDeclaration"; }
+    virtual StringView class_name() const override { return "InvalidDeclaration"; }
```

Comment:
You should probably use the `sv` suffix for string literals that your return as a StringView, saves a runtime `strlen`.

Reply:
That is not really true. The optimizer sees through the call to `strlen`. Check out this link to see what actually gets generated: https://serenityos.godbolt.org/z/vMvbMYbKd
Left side is without the use of sv, right side is with sv.